frontend/src/App.tsx

```diff
@@ -6,19 +6,35 @@ import TodoList from "./components/TodoList";
 import { Todo } from "./models/todo.model";
 
 function App() {
-  const [todos, setTodos] = useState<Todo[]>([
-    { description: "Hello world", id: "244454", isDone: false },
-    { description: "Hello Pakistan", id: "24454", isDone: false },
-    { description: "Hello GB", id: "24433", isDone: false },
-  ]);
-  const [newTodo, setNewTodo] = useState<Todo>({
-    description: "Hello world",
-    id: "244454",
-    isDone: false,
-  });
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [newTodoDescription, setNewTodoDescription] = useState("");
 
   const onChange = (name: string, value: string) => {
-    console.log(name, ":", value);
+    setNewTodoDescription(value);
+  };
+
+  const addTask = () => {
```

Comment:
Rewrite this function using `async`/`await`. This will make the function much more easy to read. Do not forget to handle errors with this approach. You can review [this](https://app.pluralsight.com/course-player?clipId=73b8560c-36dd-45cc-bdef-77cb62ec0973) video from your NodeJS course to refresh the concept.